llvm/test/CodeGen/RISCV/rvv/fixed-vector-i8-index-cornercase.ll

```diff
@@ -104,60 +104,73 @@ define <512 x i8> @two_source(<512 x i8> %a, <512 x i8> %b) {
 ; CHECK-NEXT:    addi s0, sp, 1536
 ; CHECK-NEXT:    .cfi_def_cfa s0, 0
 ; CHECK-NEXT:    andi sp, sp, -512
-; CHECK-NEXT:    vsetivli zero, 1, e8, m1, ta, ma
-; CHECK-NEXT:    vmv8r.v v24, v8
+; CHECK-NEXT:    vsetivli zero, 8, e64, m1, ta, ma
```

Comment:
I think this particularly test case is intended to hit the generic fallback.  Can you commit a test change such that the indices aren't disjoint and then rebase?